src/commands/account/list.ts

```diff
@@ -0,0 +1,37 @@
+import { Command, Flags } from "@oclif/core";
+import { yellowBright, greenBright } from "chalk";
+import { ensureSwankyProject, getSwankyConfig } from "../../lib/command-utils";
+
+export class CreateAccount extends Command {
+  static description = "List dev accounts stored in config";
+  static aliases = [`account:ls`];
+  static flags = {
+    verbose: Flags.boolean({
+      char: "v",
+    }),
+  };
+
+  static args = [];
+
+  async run(): Promise<void> {
+    await ensureSwankyProject();
+
+    const { flags } = await this.parse(CreateAccount);
+
+    const config = await getSwankyConfig();
+    this.log(`${greenBright("✔")} Stored dev accounts:`);
+    if (flags.verbose) {
+      config.accounts.forEach((account) => {
+        this.log(
+          `\t${yellowBright("Alias: ")} ${account.alias.padEnd(
+            12
+          )} ${yellowBright("Mnemonic: ")} ${account.mnemonic}`
```

Comment:
I don't think it's a good idea to log the mnemonic to the console

Reply:
I was thinking about how it shouldn't be in the in the config neither, that's why there's a warning to only store dev accounts, and this is output only with --verbose flag.
It's useful for example when you want to add those to polkadot.js app to transfer some tokens to it.
That being said, I agree, we can't trust the user not to use those in prod. I'll remove the option for now and think about how to actually prevent using stored accounts for prod deployments.